packages/data-explorer/src/views/PageHeaderView.vue

```diff
@@ -63,7 +63,9 @@ export default Vue.extend({
     ])
   },
   mounted () {
-    this.getGroupTabels({ package: this.tableMeta.package })
+    if (this.tableMeta) {
```

Comment:
the if should not be needed and would break the flow, the tableMeta should be there on mount

Reply:
Removed